system-reinstall-bootc/src/prompt.rs

```diff
@@ -82,12 +92,13 @@ pub(crate) fn get_root_key() -> Result<Option<UserKeys>> {
         prompt_user_selection(&users)?
     };
 
-    ensure!(
-        selected_users.iter().all(|x| x.user == "root"),
-        "Only importing the root user keys is supported for now"
-    );
+    let keys = selected_users
+        .into_iter()
+        .map(|user_key| user_key.authorized_keys.as_str())
+        .collect::<Vec<&str>>()
+        .join("\n");
```

Comment:
I'm fine with this for now but I do think we should link to https://lib.rs/crates/openssh-keys at some point to have a more proper validator/parser/writer

Reply:
filed https://github.com/containers/bootc/issues/1148 as a followup